docs/web-servers/apache-tips-and-tricks/modevasive-on-apache.md

```diff
@@ -145,3 +147,7 @@ DOSWhitelist 127.0.0.*
 {{< /file >}}
 
 Wildcards can be used on up to the last 3 octets if necessary. Multiple DOSWhitelist commands may be used in the configuration.
+
+## Test mod_evasive
+
+Refer to our guide on [Load Testing with Siege](/docs/tools-reference/tools/load-testing-with-siege/) to test your site's performance. Before you attempt to DDoS yourself, be aware that you risk banning your own IP. Linode does not recommend testing any server that isn't your own.
```

Comment:
This should be citing a policy otherwise probably should not phrase it like this.